### DISCUSSION TITLE
How can i found a project card from an issue id ?

### DISCUSSION THREAD

**Discussion by tassin-gauthier** [community/community#33963] [Other Feature Feedback, Questions, & Ideas]
Hello, I'm using the "issue_comment" webhook trigger in a github workflow action, and this issue is created from an organization project, so it's associated to a project card. Is it possible to get the project card id associated with the current commented issue ?

I tried with rest and graphql API and no one seems to give me the information even if it's seems closed with graphql api.

I tried this query below but 1) projectCards is an empty table on each issue (and it shouldn't) 2) even if it's working, I just want the project card associated with my issue, and not browsing every project card of the project to find the good one... any idea ?

```
query MyQuery {
  organization(login: "<my-org>") {
    projectV2(number: <my-project-id>) {
      items(first: 10) {
        edges {
          node {
            content {
              ... on Issue {
                id
                title
                projectCards(first: 10) {
                  nodes {
                    id
                    state
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}
```

My goal is pretty simple : when someone make a specific comment on an issue, it move the project card associated to a project column

**Comment by jszwedko** [comment]
I have the same use-case. My workaround was to use `addProjectV2ItemById` mutation to add the issue to the board again. If it is already there it'll just return the id of the existing card (and also leave it in whatever column it is currently in).

(I think `projectCards` is for accessing the deprecated "Projects (Classic)"; all of the new projects entities seem to have V2 in their name).
